pkg/agent/datamodel/const.go

```diff
@@ -58,14 +60,23 @@ const (
 	StandardVMType = "standard"
 )
 
+// Azure API Versions
 const (
+	APIVersionNetwork = "2018-08-01"
+)
+
+const (
+	// DefaultEnableCSIProxyWindows determines if CSI proxy should be enabled by default for Windows nodes
+	DefaultEnableCSIProxyWindows = false
 	// DefaultWindowsSSHEnabled is the default windowsProfile.sshEnabled value
 	DefaultWindowsSSHEnabled = true
 )
 
 const (
 	// AzureChinaCloud is a const string reference identifier for china cloud
 	AzureChinaCloud = "AzureChinaCloud"
+	// AzureStackCloud is a const string reference identifier for Azure Stack cloud
+	AzureStackCloud = "AzureStackCloud"
```

Comment:
will keep reading but we don't have AzureStackCloud

Reply:
This is used by replacing the env name in azure cni configuration for custom cloud.